packages/sparkling-controls/src/test.svelte

```diff
@@ -0,0 +1,184 @@
+<script lang="ts">
```

Comment:
sparkling-controls is a good place for controls with some useful names. It would be nice to have it clear how to use. Thanks.